monai/bundle/scripts.py

```diff
@@ -558,33 +562,48 @@ def _get_all_bundles_info(
     bundle_name_pattern = re.compile(r"_v\d*.")
     bundles_info: dict[str, dict[str, dict[str, Any]]] = {}
 
-    for release in releases_list:
-        if release["tag_name"] == tag:
-            for asset in release["assets"]:
-                asset_name = bundle_name_pattern.split(asset["name"])[0]
-                if asset_name not in bundles_info:
-                    bundles_info[asset_name] = {}
-                asset_version = asset["name"].split(f"{asset_name}_v")[-1].replace(".zip", "")
-                bundles_info[asset_name][asset_version] = {
-                    "id": asset["id"],
-                    "name": asset["name"],
-                    "size": asset["size"],
-                    "download_count": asset["download_count"],
-                    "browser_download_url": asset["browser_download_url"],
-                    "created_at": asset["created_at"],
-                    "updated_at": asset["updated_at"],
-                }
-            return bundles_info
+    if tag == "dev":
+        for asset in releases_list.keys():
+            asset_name = bundle_name_pattern.split(asset)[0]
+            if asset_name not in bundles_info:
+                bundles_info[asset_name] = {}
+            asset_version = asset.split(f"{asset_name}_v")[-1]
+            bundles_info[asset_name][asset_version] = {
+                "name": asset,
+                "browser_download_url": releases_list[asset]["source"],
+            }
+        return bundles_info
+    else:
+        for release in releases_list:
+            if release["tag_name"] == tag:
+                for asset in release["assets"]:
+                    asset_name = bundle_name_pattern.split(asset["name"])[0]
+                    if asset_name not in bundles_info:
+                        bundles_info[asset_name] = {}
+                    asset_version = asset["name"].split(f"{asset_name}_v")[-1].replace(".zip", "")
+                    bundles_info[asset_name][asset_version] = {
+                        "id": asset["id"],
+                        "name": asset["name"],
+                        "size": asset["size"],
+                        "download_count": asset["download_count"],
+                        "browser_download_url": asset["browser_download_url"],
+                        "created_at": asset["created_at"],
+                        "updated_at": asset["updated_at"],
+                    }
```

Comment:
```suggestion
                    bundles_info[asset_name][asset_version] = dict(asset)
```
No harm in copying the whole thing?

Reply:
Sure, just some other useless information, but updated it in the latest commit. Thanks!